src/Camera.xml

```diff
@@ -0,0 +1,114 @@
+            </enumerationValues>
+        </property>
+        <property key="height" type="integer" defaultValue="477">
+            <caption>Height</caption>
+            <category>Appearance</category>
+            <description/>
+        </property>
+        <property key="fileType" type="enumeration" defaultValue="jpeg">
+            <caption>File type</caption>
+            <category>Behaviour</category>
+             <description>Select the format in which the image is captured</description>
+            <enumerationValues>
+                <enumerationValue key="webp">webp</enumerationValue>
+                <enumerationValue key="jpeg">jpeg</enumerationValue>
+                <enumerationValue key="png">png</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="imageFilter" type="enumeration" defaultValue="none">
+            <caption>Image filter</caption>
+            <category>Appearance</category>
+             <description>Filter to be applied to the image</description>
+            <enumerationValues>
+                <enumerationValue key="grayscale">Grayscale</enumerationValue>
+                <enumerationValue key="none">Color</enumerationValue>
+                <enumerationValue key="sepia">Sepia</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="photo" type="entity" required="true">
+            <caption>Select entity</caption>
+            <category>Behaviour</category>
+            <description>This is the entity property</description>
+        </property>
+        <property key="captureButtonName" type="translatableString" required="false" >
+            <caption>Capture button</caption>
+            <category>Button captions</category>
+            <description>The class on the button that captures the image. This button should not have an action</description>
+            <translations>
+                <translation lang="en_US">Take photo</translation>
+                <translation lang="nl_NL">Maak enn foto</translation>
+            </translations>
+        </property>
+        <property key="captureButtonIcon" type="string" defaultValue="camera">
+            <caption>Capture icon</caption>
+            <category>Button captions</category>
+            <description>The name of the glyph-icon to use for capture feature</description>
+        </property>
+        <property key="switchCameraIcon" type="string" defaultValue="refresh">
+            <caption>Switch camera icon</caption>
+            <category>Button captions</category>
+            <description>The name of the glyph-icon to use for switch camera</description>
+        </property>
+        <property key="usePictureButtonIcon" type="string" defaultValue="download">
+            <caption>Use picture icon</caption>
+            <category>Button captions</category>
+            <description>This button provides the option to capture the image</description>
```

Comment:
You're describing what the button does, but the property is for its caption.

Reply:
Corrected that